### ISSUE TITLE
No documentation for jsx-child-element-spacing rule

### ISSUE THREAD

**Issue by Arcanemagus** [yannickcr/eslint-plugin-react#1634]
It seems that when `jsx-child-element-spacing` was added in #1519 there was no corresponding documentation added.

**Comment by astorije** [comment]
Also, link in [the changelog for 7.6.0](https://github.com/yannickcr/eslint-plugin-react/blob/master/CHANGELOG.md#added) points to a 404.

**Comment by astorije** [comment]
In fact, using v7.6.0 and adding `jsx-child-element-spacing: true` to the ESLint config file, I get:

```
  1:1  error  Definition for rule 'jsx-child-element-spacing' was not found  jsx-child-element-spacing
```

**Comment by pfhayes** [comment]
Thanks, I added some docs here: https://github.com/yannickcr/eslint-plugin-react/pull/1665

**Comment by ljharb** [comment]
Fixed in #1665.
